spec/mongoid/contextual/aggregable/memory_table.yml

```diff
@@ -0,0 +1,88 @@
+sets:
+  integers:
+    values:
+      - 324
+      - 4553
+      - -74
+      - 634
+      - 0
+      - 3484
+      - -3394
+      - 493
+    expected:
+      integer:
+        avg: 752.5
+        sum: 6020
+        min: -3394
+        max: 4553
+      float:
+        avg: 752.5
+        sum: 6020.0
+        min: -3394.0
+        max: 4553.0
+      big_decimal:
+        avg: !ruby/object:BigDecimal 36:0.7525e3
+        sum: !ruby/object:BigDecimal 18:0.602e4
+        min: !ruby/object:BigDecimal 18:-0.3394e4
+        max: !ruby/object:BigDecimal 18:0.4553e4
+
+  floats:
+    values:
+      - 233.132425
+      - -974.332
+      - 532.53323
+      - 0
+    expected:
+      integer:
+        avg: -52.25
+        sum: -209
+        min: -974
+        max: 532
+      float:
+        avg: -52.16658625
+        sum: -208.666345
+        min: -974.332
+        max: 532.53323
+      big_decimal:
+        avg: !ruby/object:BigDecimal 45:-0.5216658625e2
+        sum: !ruby/object:BigDecimal 27:-0.208666345e3
+        min: !ruby/object:BigDecimal 18:-0.974332e3
+        max: !ruby/object:BigDecimal 18:0.53253323e3
+
+  mixed:
+    values:
+      - 233
+      - '-974.332'
+      - ~
+      - 532.45642
+      - '-5675'
+      - 348.434
+      - 'Foobar'
+      - -3394
+      - -493
+      - ''
+      - '0 1 2 3'
+      - !ruby/object:BigDecimal 18:0.53245642e3
+      - -5675
+      - '0.434'
+      - !ruby/object:BigDecimal 18:-0.97676e5
+      - 0.434
+      - 7455.96
+      - -0.0544
+      - '   '
```

Comment:
Everything in this PR looks good except for this list. It is unclear which values are used in the calculation and which are not. For example, the numeric strings. Can you please annotate the values that aren't used with comments?

Reply:
Done. I've added `# ignored` to all unused values.

Note that strings like `"1.234"` *are* used because of the type coercion logic (i.e. the logic recently updated in MONGOID-5222).